x/service/keeper/query_service.go

```diff
@@ -0,0 +1,60 @@
+package keeper
+
+import (
+	"context"
+
+	"cosmossdk.io/store/prefix"
+	"github.com/cosmos/cosmos-sdk/runtime"
+	"github.com/cosmos/cosmos-sdk/types/query"
+
+	"github.com/pokt-network/poktroll/x/service/types"
+	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// ServiceAll queries all services.
+func (k Keeper) ServiceAll(ctx context.Context, req *types.QueryAllServiceRequest) (*types.QueryAllServiceResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "invalid request")
```

Comment:
```suggestion
		// TODO_TECHDEBT(#348): ensure consistent error convention is used.
		return nil, status.Error(codes.InvalidArgument, "invalid request")
```

Reply:
The usage of grpc error status codes is part of the error variable convention. We may choose to use them only in certain contexts but it is a degree of freedom that should be part of that conversation.